### ISSUE TITLE
Plugin not working as of Update 6.2

### ISSUE THREAD

**Issue by MikeUnseen** [SF-XIV/RolePlayersToolbox-FP-Edition#4]
Plugin disabled from 6.2 update.  Will this be updated to work for 6.2 and beyond?

**Comment by Shaece** [comment]
Go to your "_**%appdata%\XIVLauncher\installedPlugins\RoleplayersToolbox\0.3.5**_" Directory then open up the file "_**RoleplayersToolbox.json**_" and change the line "_"DalamudApiLevel": **6**,_" to "_"DalamudApiLevel": **7**,_" then open the game and Dalamud should think it's compatible with the current version to allow you to re-enable and use the add-on.

Will everything on the addon still work properly?:               - As long as nothing major changed to break any of its functions.
Will this workaround work for all future updates just by changing the number:                - Who knows.

**Comment by MikeUnseen** [comment]
Thank you, this worked for the time being.  Reports from friends that have used it said that once the buttons are added to your bar, they're good to go.  Even with the plugin not working for them after 6.2, the buttons still worked.  I'm hoping that's the case here.

**Comment by Shaece** [comment]
Yes, the glitched emotes will remain even if the add-on isn't working, can't say for sure if they get saved or not when you make configuration backups for the game though, the sleep emote can also be added to your favorite emotes list through recently used emotes when you use it but the unsnapped sit one only shows up in the recently used as a blank emote if you accidentally remove it you can place it back from the recently used emotes just can't execute it or add it to favorites from the emote list, for anyone using it to bookmark estates though might wanna consider writing those down somewhere else before it becomes unusable in the event that it does, it was called the final pull version so I wouldn't expect any compatibility updates.

**Comment by MadeByJay** [comment]
Late to the party so the above advice doesn't work - I can't install plugin as dalamud says it's outdated.

**Comment by ladyredxiv** [comment]
I am experiencing the same issue as the user above. The plugin will not install at all as Dalamud returns saying it is outdated. Are there any plans to update this for the current version of XIV, especially with the recent hotfix deployment? (6.28)

**Comment by AtomicBard104** [comment]
Redid the coding for the .json link that you place in Dalamud to update the API level to 7. Just tested it out installing for the first time and everything seems to work! Here's the link to the new copy paste html!

https://raw.githubusercontent.com/AtomicBard104/RolePlayersToolbox-FP-Edition/main/repo.json

**Comment by ladyredxiv** [comment]
omg tysm!!
